api/controller/contestants.go

```diff
@@ -67,3 +67,45 @@ func (c controllers) GetSingleContestant(w http.ResponseWriter, r *http.Request)
 
 	resp.JSON(w, http.StatusNotFound, map[string]string{"error": "id not found"})
 }
+
+func (c controllers) GetContestansConcurrently(w http.ResponseWriter, r *http.Request) {
+
+	resp := render.New()
+	query := r.URL.Query()
+
+	class := query["type"]
+	maxStr := query["items"]
+	ixwStr := query["items_per_workers"]
+
+	if len(class) == 0 {
+		returnError(resp, w, http.StatusBadRequest, errors.New("bad request"))
```

Comment:
You could be a bit more descriptive on the error returned to the use, like: `bad request: invalid type param`